### PR TITLE
Reduce redundant message wrapping code

### PR DESCRIPTION
**Pull request by ravage84**
Relates to https://github.com/cakephp/cakephp/pull/10925

### CODE REVIEW

**Comment by dereuromark** [comment]
+24 −24
Didnt save a lot of lines :)

**Comment by ravage84** [comment]
I didn't say there was a saving in code lines. 😼 
It just removes redundant code and moves it behind a single, documented &  reusable (internal) interface.

**Comment by dereuromark** [comment]
I would have just removed the array part in 4.0 ;)
But sure, :+1: sounds fine.

File: src/Console/Shell.php [hunk 1]
```diff
@@ -708,13 +698,8 @@ public function info($message = null, $newlines = 1, $level = Shell::NORMAL)
...
     public function warn($message = null, $newlines = 1)
     {
-        if (is_array($message)) {
-            foreach ($message as $k => $v) {
-                $message[$k] = '<warning>' . $v . '</warning>';
-            }
-        } else {
-            $message = '<warning>' . $message . '</warning>';
-        }
+        $messageType = 'warning';
```
Comment (dereuromark) [inline src/Console/Shell.php]: why not directly using the string in the method? The assignment seems not necessary.

**Pull request merged**